src/servers/graphql-server.ts

```diff
@@ -37,7 +37,7 @@ export const isAuthenticated = rule({ cache: "contextual" })((parent, args, ctx)
 
 export const isApiKeyAuthenticated = rule({ cache: "contextual" })(
   (_parent, _args, ctx) => {
-    return ctx.account !== null ? true : "NOT_AUTHENTICATED"
+    return ctx.account !== null ? true : "NOT_AUTHENTICATED_APIKEY"
```

Comment:
What's the context around this change?

Reply:
there are issues related to nginx in prod. and the string is `NOT_AUTHENTICATED`. so making this string unique with the previous condition will help to understand where the issue is